ilastik/applets/tracking/conservation/opConservationTracking.py

```diff
@@ -406,12 +408,37 @@ def constructFov(shape, t0, t1, scale=[1, 1, 1]):
                     coordinatesForIds = {}
                     
                     pool = RequestPool()
+                    poolIsNotEmpty = False
+                    
                     for objectId in objectIds:
-                        pool.add(Request(partial(mergerResolver.getCoordinatesForObjectId, coordinatesForIds, labelImage[0, ..., 0], objectId)))                   
-                    pool.wait()                
+                        node = (timestep, objectId)
+                         
+                        withCoords = False
+                         
+                        if originalGraph.hasNode(node) and 'value' in originalGraph._graph.node[node] and originalGraph._graph.node[node]['value'] > 1:
+                            withCoords = True
+                         
+                        if not withCoords:
+                            for edge in originalGraph._graph.out_edges(node):
+                                neighbor = edge[1]                        
+                                if  originalGraph.hasNode(neighbor) and 'value' in originalGraph._graph.node[neighbor] and  originalGraph._graph.node[neighbor]['value'] > 1:
+                                    withCoords = True
+                                    break
+                                                 
+                        if withCoords:    
+                            pool.add(Request(partial(mergerResolver.getCoordinatesForObjectId, coordinatesForIds, labelImage[0, ..., 0], objectId)))
+                            poolIsNotEmpty = True                   
+                    
+                    if poolIsNotEmpty:
```

Comment:
Tiny note: `RequestPool` has a `__len__` method, so you could simplify this logic with `if len(pool):`.  (**Edit:** I see that this code is removed in a later commit anyway, so there's nothing to do here...)